pkg/util/span/frontier.go

```diff
@@ -125,6 +125,13 @@ func MakeFrontier(spans ...roachpb.Span) (*Frontier, error) {
 // Each span timestamp initialized at specified start time.
```

Comment:
@miretskiy Could I get your blessing on this low impact change here?

Reply:
THis change is :lgtm: